DescendantBooks/po/it-local.po

```diff
@@ -0,0 +1,739 @@
+msgid "unknown"
+msgstr "sconosciuto"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1015
+#, python-format
+msgid "Children of %(mother_name)s and %(father_name)s"
+msgstr "Figlio di %(mother_name)s e %(father_name)s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1073
+#, python-format
+msgid "Notes for %(mother_name)s and %(father_name)s:"
+msgstr "Annotazioni di %(mother_name)s e %(father_name)s:"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1096
+#: DescendantBooks/DetailedDescendantBookReport.py:1119
+#, python-format
+msgid "More about %(mother_name)s and %(father_name)s:"
+msgstr "Altro su %(mother_name)s e %(father_name)s:"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1201
+#, python-format
+msgid "Notes for %s"
+msgstr "Annotazioni per %s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1214
+#: DescendantBooks/DetailedDescendantBookReport.py:1233
+#: DescendantBooks/DetailedDescendantBookReport.py:1247
+#: DescendantBooks/DetailedDescendantBookReport.py:1271
+#, python-format
+msgid "More about %(person_name)s:"
+msgstr "Altro a proposito di %(person_name)s:"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1221
+#, python-format
+msgid "%(name_kind)s: %(name)s%(endnotes)s"
+msgstr "%(name_kind)s: %(name)s%(endnotes)s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1260
+msgid "Address: "
+msgstr "Indirizzo: "
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1282
+#, python-format
+msgid "%(type)s:"
+msgstr "%(type)s:"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1287
+#, python-format
+msgid " %(value)s%(endnotes)s"
+msgstr " %(value)s%(endnotes)s"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1351
+msgid "Henry numbering"
+msgstr "Numerazione di Henry"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1352
+msgid "d'Aboville numbering"
+msgstr "Numerazione d'Aboville"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1354
+msgid "Record (Modified Register) numbering"
+msgstr "Numerazione a record (registro modificato)"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1364
+msgid "Page break between generations"
+msgstr "Interruzione di pagina tra due generazioni"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1366
+msgid "Whether to start a new page after each generation."
+msgstr "Indica se iniziare una nuova pagina dopo ogni generazione."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1369
+msgid "Page break before end notes"
+msgstr "Interruzione di pagina prima delle annotazioni"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1371
+msgid "Whether to start a new page before the end notes."
+msgstr "Indica se iniziare una nuova pagina prima delle annotazioni."
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1378
+msgid "Content"
+msgstr "Contenuto"
+
+#: DescendantBooks/DetailedDescendantBookReport.py:1380
+msgid "Use callname for common name"
+msgstr "Usa il soprannome come nome comune"
```

Comment:
I used pseudonimo for callname; soprannome is used for nickname.
We can argue whether the two terms should be switched, but for now they should be consistent.